### DISCUSSION TITLE
Grafana impot alert rule

### DISCUSSION THREAD

**Discussion by wooyoungAhn** [NetApp/harvest#2227] [Q&A] [answered]
Hello everyone 
I am trying add alert rule in Harvest. Does anyone have an example alert rule template like a [dashboard](https://github.com/NetApp/harvest/tree/main/grafana)?

Do alerts generally depend only on Unified Manager?

**Comment by rahulguptajss** [comment, marked as answer]
@wooyoungAhn See if this helps. https://netapp.github.io/harvest/latest/prometheus-exporter/#prometheus-alerts

**Reply by wooyoungAhn** [reply]
Hello
sorry i'm late 
Thank you for your help.
I will try it.
